frontend/micro-ui/web/micro-ui-internals/packages/modules/pt/src/pages/citizen/Create/Attachments.js

```
@@ -0,0 +1,151 @@
+    >
+      <path d="M21.44 11.05L12.97 19.51a5.25 5.25 0 01-7.42-7.42l8.48-8.48a3.5 3.5 0 014.95 4.95l-8.49 8.48a1.75 1.75 0 01-2.47-2.47l7.78-7.78" />
+    </svg>
+  );
+
+  const renderFileInput = (id, label, isRequired = false) => (
+    <div style={styles.fileBox}>
+      <div style={styles.iconBox}>{renderSvg()}</div>
+      <div style={styles.labelArea}>
+        <label style={styles.fileLabel}>
+          {t(label)} {isRequired && <span style={{ color: "red" }}>*</span>}
+        </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)